### PR TITLE
feat: add Boba BNB Testnet

### PR DESCRIPTION
**Pull request by boyuan-chen**
### Testing

```
~/.hyperlane/chains/boba main* ❯ hyperlane send message --relay                                                                    2m 2s 01:31:51 PM
Hyperlane CLI
? Select network type Testnet
? Select the origin chain holesky
? Select network type Testnet
? Select the destination chain bobabnbtestnet
Running pre-flight checks for chains...
✅ Chains are valid
✅ Signer is valid
✅ Balances are sufficient
Dispatching message
Pending https://holesky.etherscan.io/tx/0x43fabfdf0e33314114abb90aeae0087a7db619d83712e385fda7ab621480a799 (waiting 1 blocks for confirmation)
Sent message from holesky to 0x2641c606fc26d77D4430c443E95acbf1bBA324EE on bobabnbtestnet.
Message ID: 0x5449d76c47baf6158158e27f11d5b87af38c61f1cdc532d05fcfc4e691123dc7
Message:
    parsed:
      version: 3
      nonce: 417
      origin: 17000
      sender: "0x000000000000000000000000d3edfbd01829c5a97b94b20f82dc17088cc00031"
      destination: 9728
      recipient: "0x0000000000000000000000002641c606fc26d77d4430c443e95acbf1bba324ee"
      body: "0x48656c6c6f21"
      originChain: holesky
      destinationChain: bobabnbtestnet
    id: "0x5449d76c47baf6158158e27f11d5b87af38c61f1cdc532d05fcfc4e691123dc7"
    message: "0x03000001a100004268000000000000000000000000d3edfbd01829c5a97b94b20f8\
      2dc17088cc00031000026000000000000000000000000002641c606fc26d77d4430c443e95acb\
      f1bba324ee48656c6c6f21"

Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Attempting self-relay of message
Preparing to relay message 0x5449d76c47baf6158158e27f11d5b87af38c61f1cdc532d05fcfc4e691123dc7
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
Unhandled error case in combined provider error handler
{
  message: {
    id: '0x5449d76c47baf6158158e27f11d5b87af38c61f1cdc532d05fcfc4e691123dc7',
    message: '0x03000001a100004268000000000000000000000000d3edfbd01829c5a97b94b20f82dc17088cc00031000026000000000000000000000000002641c606fc26d77d4430c443e95acbf1bba324ee48656c6c6f21',
    parsed: {
      version: 3,
      nonce: 417,
      origin: 17000,
      sender: '0x000000000000000000000000d3edfbd01829c5a97b94b20f82dc17088cc00031',
      destination: 9728,
      recipient: '0x0000000000000000000000002641c606fc26d77d4430c443e95acbf1bba324ee',
      body: '0x48656c6c6f21'
    }
  },
  metadata: '0x'
} Relaying message 0x5449d76c47baf6158158e27f11d5b87af38c61f1cdc532d05fcfc4e691123dc7
Pending https://testnet.bobascan.com/tx/0x90464748dd59a2f14385a849a2819b587ca2d4fc20008bcc56a485b926a117ba (waiting 1 blocks for confirmation)
Message was self-relayed!
```

### CODE REVIEW

**Comment by JunayedMh** [comment]
You didn’t add the logo.svg file...add the chain logo on your folder

**Comment by boyuan-chen** [comment]
> You didn’t add the logo.svg file...add the chain logo on your folder

Hey @JunayedMh fixed.

File: chains/bobabnbtestnet/logo.svg
Comment (atuser14) [inline chains/bobabnbtestnet/logo.svg:1]: submit the logo again running through https://jakearchibald.github.io/svgomg/

**Comment by atuser14** [comment]
you need to add a changeset file...in the changeset file rename the patch with minor

**Comment by JunayedMh** [comment]
you need to add a changeset file.

**Review by xeno097 — CHANGES_REQUESTED** [review]
Hey @boyuan-chen thank you for your PR to the hyperlane registry! As @JunayedMh said the changeset is missing in the pr please add one with the `minor` version bump

**Comment by boyuan-chen** [comment]
> Hey @boyuan-chen thank you for your PR to the hyperlane registry! As @JunayedMh said the changeset is missing in the pr please add one with the `minor` version bump

Added

**Review by xeno097 — APPROVED** [review]

**Pull request merged**